ArjunaJTS/jtax/pom.xml

```diff
@@ -145,11 +149,281 @@
       <artifactId>h2</artifactId>
       <version>${version.com.h2database}</version>
       <scope>test</scope>
-    </dependency>    
+    </dependency>
+
   </dependencies>
 
   <profiles>
     <profile>
+      <id>j9</id>
+      <activation>
+        <jdk>[1.9,)</jdk>
+      </activation>
+      <properties>
+        <version.compiler.plugin>3.6.0</version.compiler.plugin>
+      </properties>
+      <dependencies>
+        <dependency>
+          <groupId>org.jboss.narayana.jts</groupId>
+          <artifactId>idlj-idl</artifactId>
+          <version>${project.version}</version>
+        </dependency>
+        <dependency>
+          <groupId>org.jboss.narayana.jts</groupId>
+          <artifactId>idlj-idl-tests</artifactId>
+          <version>${project.version}</version>
+          <scope>test</scope>
+        </dependency>
+        <dependency>
+          <groupId>org.jboss.narayana.jts</groupId>
+          <artifactId>${java.corba.patch.name}</artifactId>
+          <version>${project.version}</version>
+        </dependency>
+      </dependencies>
+
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-dependency-plugin</artifactId>
+            <version>2.10</version>
```

Comment:
@mmusgrov - same question regarding whether version should be in pom.xml in root